lib/ansible/modules/system/group.py

```diff
@@ -53,6 +53,12 @@
         type: bool
         default: no
         version_added: "2.6"
+    non_unique:
+        description:
+            - Optionally when used with the -g option, this option allows to change the group ID to a non-unique value.
```

Comment:
Rather than mentioning the `-g` option, which is abstracted away by this module, mention the parameter name instead.

It also seems like this doesn't do anything unless `gid` is also specified. Should this option required `gid`?
```suggestion
            - This option allows to change the group ID to a non-unique value. Requires C(gid).
```

Reply:
I totally agree, your suggestion is much better than the existing description. I will update my PR.
I copied this description from user module, should we change that too ?

Reply:
> I copied this description from user module, should we change that too ?

We can address that in a separate PR.

Reply:
> Rather than mentioning the `-g` option, which is abstracted away by this module, mention the parameter name instead.
> 
> It also seems like this doesn't do anything unless `gid` is also specified. Should this option require `gid`?
> 
> Suggested change 
>             - Optionally when used with the -g option, this option allows to change the group ID to a non-unique value.
>             - This option allows to change the group ID to a non-unique value. Requires C(gid).

Done